lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -92,6 +92,82 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
                                                        operand, vApproximate);
         return success();
       });
+  patterns.onOp(
+      "GridSample", 20,
+      [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+        Torch::ValueTensorType resultType;
+        Value input;
+        Value grid;
+        std::string mode;
+        Value interpolationMode;
+        std::string padding;
+        Value paddingMode;
+        int64_t align;
+        Value alignCorners;
+
+        if (binder.tensorOperandAtIndex(input, 0) ||
+            binder.tensorOperandAtIndex(grid, 1) ||
+            binder.tensorResultType(resultType))
+          return failure();
+
+        auto inputTensorType = input.getType().cast<Torch::ValueTensorType>();
+        ArrayRef<int64_t> inputShape = inputTensorType.getSizes();
+        unsigned inputRank = inputShape.size();
+        auto gridTensorType = grid.getType().cast<Torch::ValueTensorType>();
+        ArrayRef<int64_t> gridShape = gridTensorType.getSizes();
+        unsigned gridRank = gridShape.size();
+
+        if (inputRank != 4) {
+          return rewriter.notifyMatchFailure(binder.op,
+                                             "only input rank 4 supported");
+        }
+        if (gridRank != 4) {
+          return rewriter.notifyMatchFailure(binder.op,
+                                             "only grid rank 4 supported");
+        }
```

Comment:
You can drop the opening / closing brackets as this one just one line.

Reply:
cool yes have couple of those